packages/rpc/src/methods/account.ts

```diff
@@ -1,7 +1,7 @@
 import { Network } from '@mintbase-js/sdk';
 import { RPC_OPTIONS, requestFromNearRpc } from '../util';
 
-export const accountExists = async (accountId: string, network?: Network, rpc?: RPC_OPTIONS): Promise<boolean> => {
+export const accountExists = async (accountId: string, network?: Network, rpc?: RPC_OPTIONS, rpcUrl?: string): Promise<boolean> => {
```

Comment:
Just a note that: 
The fact that we had to change the input parameters over so many files suggest that we should have a type interface for input arguments to our functions.

```
interface networkArgs {
   rpcUrl: string;
   network: Network;
}
```

So then we would only have to change it in one place.